src/polyswarmd/__init__.py

```diff
@@ -37,16 +37,14 @@
 
 from polyswarmd.eth import misc
 from polyswarmd.utils import bool_list_to_int, int_to_bool_list
-from polyswarmd.artifacts.artifacts import artifacts, MAX_ARTIFACT_SIZE_REGULAR, MAX_ARTIFACT_SIZE_ANONYMOUS
+from polyswarmd.artifacts.artifacts import artifacts, FALLBACK_MAX_ARTIFACT_SIZE
 from polyswarmd.balances import balances
 from polyswarmd.bounties import bounties
 from polyswarmd.relay import relay
 from polyswarmd.offers import offers
 from polyswarmd.staking import staking
 from polyswarmd.websockets import init_websockets
 
-app.config['MAX_CONTENT_LENGTH'] = MAX_ARTIFACT_SIZE_REGULAR * 256
```

Comment:
@sbnetops are we settings `MAX_CONTENT_LENGTH` in AI anywhere? Don't remember where we landed on that, but it should match what's happening on `polyswarmd`.

Reply:
We are now, we do that here:
https://github.com/polyswarm/artifact-index/blob/f01b93e248659e9df0338f316f1b17da8a400054/src/artifact_index/settings.py#L72
It is being set to 128Mb at the moment. It validates the size here:
https://github.com/polyswarm/artifact-index/blob/f01b93e248659e9df0338f316f1b17da8a400054/src/artifact_index/views/utils/decorators.py#L204